sui/deploy-contract.js

```diff
@@ -0,0 +1,76 @@
+const { saveConfig, printInfo } = require('../evm/utils');
+const { Command, Argument } = require('commander');
+const { publishPackage, updateMoveToml } = require('@axelar-network/axelar-cgp-sui/scripts/publish-package');
+
+const { addBaseOptions } = require('./cli-utils');
+const { getWallet, printWalletInfo } = require('./sign-utils');
+const { loadSuiConfig, findPublishedObject } = require('./utils');
+
+// Add more contracts here to support more modules deployment
+const contractMap = {
+    gasService: {
+        packageName: 'gas_service',
+        contractName: 'GasService',
+        displayName: 'Gas Service',
+        chainConfigKey: 'axelar_gas_service',
```

Comment:
this is redundant since we should use GasService for the key. Let's just set the `packageName` in the config itself (it can be provided as an arg override too). I really don't like having multiple configs that we have to maintain.